web/cobrands/sass/_base.scss

```diff
@@ -1656,7 +1699,7 @@ a:hover.rap-notes-trigger {
 // and make the map full screen to reduce distractions. JavaScript also
 // tweaks the text content of some of the map-related elements, to make
 // it more "app-like".
-.mobile-reporting-map {
+.mobile-reporting-map, .report-fullscreen {
```

Comment:
I like how these share a lot, I wonder if we can clear it up a bit to split it out a bit more. So we'd have:
* one class that shows the map full screen, `map-fullscreen` perhaps;
* one class that 'locks' it, that's `only-map`;
* one class that does page-specific thing, so `mobile-reporting-map` (perhaps renamed something like map-reporting?) and its pan zoom top.

Then the report page would have `map-fullscreen only-map`, and the around page would have `map-fullscreen only-map map-reporting` (where it currently has mobile-reporting-map only-map). Does that make sense, or not worth it?